sys/net/include/etx_beaconing.h

```diff
@@ -94,25 +97,49 @@ typedef struct __attribute__((packed)) etx_probe_t {
 } etx_probe_t;
 
 typedef struct etx_neighbor_t {
-    ipv6_addr_t addr;           //The address of this node
-    uint8_t     tx_cur_round;   //The indicator for receiving a packet from this candidate this round
-    uint8_t     packets_tx[ETX_WINDOW]; //The packets this node has transmitted TO ME
-    uint8_t     packets_rx;     //The packets this node has received FROM ME
-    double      cur_etx;        //The currently calculated ETX-value
-    uint8_t     used;           //The indicator if this node is active or not
+    ipv6_addr_t addr;           /** The address of this node */
+    uint8_t     tx_cur_round;   /** The indicator for receiving a packet from this candidate this round */
+    uint8_t     packets_tx[ETX_WINDOW]; /** The packets this node has transmitted TO ME */
+    uint8_t     packets_rx;     /** The packets this node has received FROM ME */
+    double      cur_etx;        /** The currently calculated ETX-value */
+    uint8_t     used;           /** The indicator if this node is active or not */
 } etx_neighbor_t;
 
 //prototypes
+
+ /**
+* This function is used to initialise and accepts one parameter . 
```

Comment:
You don't need to write "This function is used to" on every function. You don't need to write how many arguments a function takes, it can be inferred from the function definition and the auto-generated documentation.